compiler/infra/OMRCfg.hpp

```diff
@@ -161,8 +174,27 @@ class CFG
    void removeStructureSubGraphNodes(TR_StructureSubGraphNode *node);
 
    void addEdge(TR::CFGEdge *e);
-   TR::CFGEdge *addEdge(TR::CFGNode *f, TR::CFGNode *t, TR_AllocationKind = heapAlloc);
-   void addExceptionEdge(TR::CFGNode *f, TR::CFGNode *t, TR_AllocationKind = heapAlloc);
+
+   /**
+    * @deprecated
+    * Please specify where edges are to be allocated during CFG initialization.
+    * E.g.,
+    *     OMR::CFG(c, m, TR::Region &region)
+    * and use the following method to add edges to that region.
+    *     addEdge(TR::CFGNode *f, TR::CFGNode *t);
+    */
+   TR::CFGEdge *addEdge(TR::CFGNode *f, TR::CFGNode *t, TR_AllocationKind heapAlloc);
+   TR::CFGEdge *addEdge(TR::CFGNode *f, TR::CFGNode *t);
```

Comment:
Can you please provide doxygen for the non-deprecated APIs which people should be using long term?

Reply:
Will do.